arangod/GeneralServer/ServerSecurityFeature.cpp

```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2016 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Steemann
+////////////////////////////////////////////////////////////////////////////////
+
+#include "GeneralServer/ServerSecurityFeature.h"
+#include "Logger/Logger.h"
+#include "ProgramOptions/ProgramOptions.h"
+#include "ProgramOptions/Section.h"
+#include "Utils/ExecContext.h"
+
+using namespace arangodb;
+using namespace arangodb::basics;
+using namespace arangodb::options;
+
+ServerSecurityFeature::ServerSecurityFeature(application_features::ApplicationServer& server)
+    : ApplicationFeature(server, "ServerSecurity"),
+      _enableFoxxApi(true),
+      _enableFoxxStore(true) {
+  setOptional(false);
+  startsAfter("ServerPlatform");
+}
+
+void ServerSecurityFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
+  options->addSection("foxx", "Configure Foxx");
+  options->addOption("--foxx.api", "enables Foxx management REST APIs",
+                     new BooleanParameter(&_enableFoxxApi))
+                     .setIntroducedIn(30500);
```

Comment:
this is not true anymore.
```suggestion
                     .setIntroducedIn(30408).setIntroducedIn(30500);
```